src/_minimal/components/settings/settings-structure-estimation.ts

```diff
@@ -35,6 +35,19 @@ export const notificationsSection: ConfObj[] = [
     },
     component: SettingsGeneral,
   },
+  {
+    key: 'notificationsSticky',
+    title: () =>
+      `${api.storage.lang('settings_Notifications_Sticky')} ${api.storage.lang(
```

Comment:
This will not work in all languages. Better use "sticky notifications" as translation string